### PR TITLE
Created tasks

### PR DESCRIPTION
**Pull request by KorotkevichIvan**

### CODE REVIEW

File: homework_4/task_3.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+x = {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+y = list(x.keys())
+for i in range(len(y)):
+    y[i] += str(len(y[i]))
+a = x.values()
+print(x)
+print({k:v for k, v in zip(y, a)})
+# не знаю как через while
```
Comment (svyatkin-aleks) [inline homework_4/task_3.py:8]: user_dict = {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
new_dict = {}
list_key = tuple(user_dict.keys())
list_values = tuple(user_dict.values())
print(user_dict)
for n in range(len(user_dict)):
    new_dict[list_key[n] + str(len(list_key[n]))] = list_values[n]
    n += 1
print(new_dict)

File: homework_4/task_4.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+num = input('Введите числа через пробел: ').split()
+num += num[0]
+for i in range(len(num) - 1):
+    num[i] = num[i + 1]
+del num[-1]
+print(num)
```
Comment (svyatkin-aleks) [inline homework_4/task_4.py:6]: list_1 = [1, 2, 3, 4, 5, 6]
list_1.append(list_1.pop(0))

File: homework_4/task_3.py (hunk 1 above)
Comment (svyatkin-aleks) [reply homework_4/task_3.py:8]: n = 0
while len(new_dict) < len(user_dict):
    new_dict[list_key[n] + str(len(list_key[n]))] = list_values[n]
    n += 1
print(new_dict)

**Review by svyatkin-aleks — APPROVED** [review]

File: homework_4/task_4.py (hunk 2 above)
Comment (KorotkevichIvan) [reply homework_4/task_4.py:6]: Да, но по условию надо был через циклы

**Pull request merged**